packages/webapp/src/components/Navigation/styles.module.scss

```diff
@@ -20,4 +20,5 @@
   flex-grow: 1;
   min-height: 100vh;
   min-height: -webkit-fill-available;
+  overflow: hidden;
```

Comment:
This is to prevent the side menu bar being pushed off to the left (and separating from its collapse icon), which we have seen on a few screens at tablet width:

https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/1f73c09b-0161-4f3b-bb55-f4147f3bcd6a

In this component's final state it's not as pronounced as the video above, but I think it needs to be prevented in all cases.